### PR TITLE
Updates to the submission and data access guides

### PR DESCRIPTION
**Pull request by jmenglund**
I have made some changes to the guides after discussing with @mattiasstromberg. 

This pull request contains:

* clarifications to some steps in the guides and
* new name and text for step 3 in the submission guide so that it aligns with new procedure at Central EGA

### CODE REVIEW

**Review by shreyasshivakumara — APPROVED** [review]
Looks good! :)

File: _includes/data-access-table.html [hunk 1]
```diff
@@ -61,24 +61,27 @@
...
+    Once the DAC has accepted your data access request and an authorisation
+    grant has been received by the SDA Helpdesk, you will recieve instructions
+    from the SDA Helpdesk on how to access the data.
   </div>
 
   <div class="steps">4. Download and decrypt files</div>
   <div>
-    Download and decrypt the files according to the instructions provided 
-    by the SDA Helpdesk.
+    Download and decrypt the files according to the instructions provided by
```
Comment (dbampalikis) [inline _includes/data-access-table.html]: Should we include something about decrypting in a secure environment?

**Review by dbampalikis — APPROVED** [review]
Added one comment. Otherwise it looks good

File: _includes/data-access-table.html (hunk 1 above)
Comment (shreyasshivakumara) [reply _includes/data-access-table.html]: I have added that in the GitHub download and decrypt instructions.

Comment (jmenglund) [reply _includes/data-access-table.html]: Yes, that would be good. @shreyasshivakumara has prepared a text, but I don't think he has added it yet to https://github.com/NBISweden/EGA-SE-user-docs

Comment (shreyasshivakumara) [reply _includes/data-access-table.html]: Yes! It will be updated in 20 minutes.

**Review by norling — APPROVED** [review]
Looks good! feel free to merge once dimi's comment is resolved.

**Review by kostas-kou — APPROVED** [review]
Looks good

File: _includes/data-access-table.html (hunk 1 above)
Comment (jmenglund) [reply _includes/data-access-table.html]: Link to the SDA-specific instructions has now been added

**Pull request merged**